testcases/replicaset/replicaset.robot

```diff
@@ -0,0 +1,7 @@
+*** Settings ***
+Resource          ./replicaset_kw.robot
+
+*** Test Cases ***
+Replicaset test case example
+    [Tags]    replicaset
```

Comment:
For
> All new testcases tagged as prerelease along other tags to exclude it from execution until released on PyPI
just add **prerelease**

Reply:
> **prerelease**
I added tags prerelease and could you kindly help me in generate docs and coverage